src/components/Menu/__snapshots__/Menu.test.tsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`<Menu /> Default story renders snapshot 1`] = `
   <button
     aria-controls="headlessui-menu-items-2"
     aria-expanded="true"
-    aria-haspopup="true"
+    aria-haspopup="menu"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup (they coulda left this as `true` but are now supporting the more specific value)